dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/TrackerController.java

```diff
@@ -124,6 +105,56 @@ public void postJsonTracker( HttpServletRequest request, HttpServletResponse res
                     .build() ) );
     }
 
+    @PostMapping( value = "", consumes = MediaType.APPLICATION_JSON_VALUE, params = { "async=false" } )
```

Comment:
we can decide whether use it like 
```
/tracker?async=false
```
or
```
/tracker/async
```
the latter was suggested by @luciano-fiandesio . Honestly I have no personal preference over the one or the other